### ISSUE TITLE
Login issue once exit the application and login again to the application.

### ISSUE THREAD

**Issue by mac24079** [ABenchM/abm-frontend#212]
Whenever we log in to the ABM application once after -  without log out the application, following attached below error appears which should not be the case.

![image](https://user-images.githubusercontent.com/23357897/48666416-a0abc700-eac1-11e8-8fdf-7fef96c02bc6.png)

Once, a user exits the application and closes the terminal and logs in the above error should not appear.

**Comment by mac24079** [comment]
![403-error](https://user-images.githubusercontent.com/23357897/49082984-8903d980-f24b-11e8-89fe-9bd210e97c82.png)

**Comment by ankur-1989** [comment]
@mac24079  
When are you planning to do it ? 
if it is going to take a longer then let me know.

**Comment by ankur-1989** [comment]
We need to make one change here 
The application should log out automatically after 4 hrs.

**Comment by mac24079** [comment]
@ankur-1989 

I have included all the changes and also, the new change suggested by you that "The application should log out automatically after 4 hrs.".

I will raise the merge request for the same.

**Comment by mac24079** [comment]
Hello @ankur-1989 

I have fixed the changes for the same both in the front-end and back-end part. Kindly preview the changes and let me know if there is something more needed. Both merge requests can be found after the end of text.

What I did additional to your suggested change is that the session should log out after 4 hours is that before the last two minutes a message box will give a user an alert that - "Your session will expire in ..... seconds, so it will help the user to know that there session will expire soon". Both the branches are needed to test the descriptive issue.

Front-end merge request:
https://github.com/ABenchM/abm-frontend/pull/223.

Back-end merge request:
https://github.com/ABenchM/abm/pull/274.

Thanks.
